episodes/03-audience.md

```diff
@@ -22,48 +19,142 @@ adapted to place an emphasis on assessing learning during a workshop:
+   [mental model][it-mental-models] learners will have 
+   of the lesson domain when they arrive. 
+   This can enable you to make progress quickly
+   by 
+   a) working to help learners recall (_activate_) that prior knowledge,
+   b) building on the conceptual understanding they already have and, 
+   c) perhaps most importantly,
+   giving you some idea of what _misconceptions_ they might arrive with.
+   It is vital that misconceptions are identified and corrected early on,
+   before learners try to incorporate new knowledge into a broken mental model.
+2. People at different stages of this process need to be taught differently.
+   For example, novices will learn more from lessons that include worked examples 
+   and are more tutorial-like i.e. focused on a specific task, 
+   with step-by-step explanations of the process,
+   but without a lot of extra information that is not directly relevant. 
+   However, the same approach may actually hinder learning for competent practitioners 
+   who may be distracted by a step-by-step explanation 
+   of something they already have the prior knowledge to understand on their own.
+   For learners at this level of expertise, 
+   lessons which include activities offering learners 
+   the freedom to explore options and develop their own solutions,
+   are likely to be more effective.
+
+### Motivation
+
+Furthermore, your lesson will be more effective if 
+it aligns with the motivations of the target audience. 
+Understanding the wants and needs of your target audience, 
+what they know already and what kinds of problems they want to solve, 
+will help you design a lesson that learners can see the value in.
+It will give them the impression that taking the lesson will be worthwhile 
+(called _positive expectancies_ in the literature). 
+
+We will look more at strategies to establish value and build positive expectancies
+in the next chapter.
+
+### Be Specific
+
+It can be tempting to identify a target audience only in vague terms,
+for example by writing that a lesson is aimed at 
+"PhD students" or "early career researchers".
+However,
+taking the time to focus on real people, 
+or imagined personae, 
+who represent your target audience will help you take time to consider 
+the various aspects that can influence how much someone will learn from your lesson.
+It will also help you notice when the assumptions you are making about your target audience 
+are unreasonable.
+
+Most of all, it will help you stay connected to the fact that 
+_you are not your learners_:
+they will arrive at the lesson with different priorities, 
+interests, and challenges than your own.
 
 ::::::::::::::::::::::::::::::::::::::  challenge
 
 ## Exercise: thinking about target audience (15 minutes total for both parts)
 
 Part 1 (5 minutes): think about a member of the target audience for your lesson,
-and answer the following questions:
+and answer the following questions in the context of your lesson topic:
```

Comment:
might take more than 5 min to answer all these questions, maybe suggest a subset?

Reply:
You're right. I'll switch it back to only four questions, with a note that the other questions are a good homework exercise for those who want to spend more time thinking about their target audience.